clustermesh-apiserver/users_mgmt.go

```diff
@@ -34,6 +34,11 @@ var usersManagementCell = cell.Module(
 	cell.Invoke(registerUsersManager),
 )
 
+var usersManagementControllerGroup = controller.ControllerGroup{
+	Name:                  "clustermesh-users-management",
+	MetricsEnabledDefault: false,
```

Comment:
I'd be personally in favor of dropping the `MetricsEnabledDefault` parameter here, directly using the `controller-group-metrics` flag (and the corresponding helm setting which I suppose should be added) to enable the default ones. That would make it clear for users which are the enabled metrics (and allow different defaults for different components) as well as allow to simplify the creation here to something like `= controller.NewGroup("clustermesh-users-management")`.